src/components/ContactsList/ContactsList.jsx

```diff
@@ -1,66 +1,117 @@
-import React from 'react'
+import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 import { sortLastNameFirst, buildLastNameFirst } from './'
 import ContactsEmptyList from './ContactsEmptyList'
 import ContactRow from './ContactRow'
+import ContactHeaderRow from './ContactHeaderRow'
+import { Query } from 'cozy-client'
+import ContactCardModal from '../Modals/ContactCardModal'
+import _ from 'lodash'
```

Comment:
It's preferable to import only the functions we need from lodash, rather than the whole package. So something like:

```suggestion
import find from 'lodash/find'
```